src/Umbraco.Cms.Integrations.SEO.GoogleSearchConsole.UrlInspectionTool/App_Plugins/UmbracoCms.Integrations/SEO/GoogleSearchConsole/URLInspectionTool/js/url-inspection-tool.controller.js

```diff
@@ -0,0 +1,115 @@
+﻿function urlInspectionToolController(editorState, notificationsService, umbracoCmsIntegrationsGoogleSearchConsoleUrlInspectionToolResource) {
+    var vm = this;
+
+    vm.loading = false;
+    vm.showResults = false;
+    vm.oauthConfiguration = {};
+    vm.inspectionResult = {};
+
+    // get available cultures for current node
+    vm.currentCultures = editorState.current.urls.map(p => p.culture);
+
+    // build default url inspection object
+    vm.siteUrl = location.href.slice(0, location.href.indexOf("umbraco") - 1);
```

Comment:
You might have to find a way to do this by position, as it's not 100% sure that the root path is `/umbraco/` (I believe you can configure something else).

Don't forget also that the user facing website URL may be different to that used for the back-office.

Reply:
Updated. Included a dropdownlist to handle the different URLs, and the language code will be automatically picked by matching with the selected URL.